specificatie/genereervariant/openapi.yaml

```diff
@@ -4355,21 +3970,19 @@ components:
     Beslag_links:
       type: object
       properties:
-        beslaglegger:
-          $ref: '#/components/schemas/HalLink'
-        stukken:
+        beslagleggers:
           type: array
           items:
             $ref: '#/components/schemas/HalLink'
-        isGebaseerdOpStukdeel:
+        stukken:
           type: array
           items:
             $ref: '#/components/schemas/HalLink'
-        isVermeldInStukdelen:
+        isGebaseerdOpStukdeel:
           type: array
           items:
             $ref: '#/components/schemas/HalLink'
-        zakelijkGerechtigden:
```

Comment:
Ik zie `zakelijkGerechtigden` hier verwijderd. Ik weet niet of dit de bedoeling is, maar ik zie deze **_niet_** verwijderd in de niet-genereervariant specificatie. Dus dat zou ik even gelijktrekken.

Reply:
Deze heb ik ook verwijderd in de niet-genereervariant.

Reply:
De genereervariant wordt gegenereerd  door het resolven (externe $refs worden naar lokaal gemaakt) van de niet-genereervariant. Aanpassingen hoeven dus alleen in de niet-genereervariant worden doorgevoerd.